csp-mixin/signals/gcpce.libsonnet

```diff
@@ -0,0 +1,119 @@
+local commonlib = import 'common-lib/common/main.libsonnet';
+function(this)
+  {
+    local s = self,
+    filteringSelector: 'job!=""',
+    groupLabels: ['job', 'project_id'],
+    instanceLabels: ['instance_name'],
+    aggLevel: 'instance',
+    discoveryMetric: {
+      stackdriver: 'stackdriver_gce_instance_compute_googleapis_com_instance_cpu_utilization',
+    },
+    signals: {
+      cpuUtilization: {
+        name: 'CPU Utilization',
+        description: 'Fractional utilization of allocated CPU on this instance.',
+        type: 'raw',
+        unit: 'percentunit',
+        sources: {
+          stackdriver: {
+            expr: 'stackdriver_gce_instance_compute_googleapis_com_instance_cpu_utilization{%(queriesSelector)s}',
+            legendCustomTemplate: '{{instance_name}}',
+          },
+        },
+      },
+      cpuUsageTime: {
+        name: 'Cpu usage time',
+        description: 'CPU usage, in seconds. For Container-Optimized OS, or Ubuntu running GKE.',
+        type: 'counter',
+        unit: 'seconds',
+        aggFunction: 'avg',
+        sources: {
+          stackdriver: {
+            expr: 'stackdriver_gce_instance_compute_googleapis_com_instance_cpu_usage_time{%(queriesSelector)s}',
+            legendCustomTemplate: '{{instance_name}}',
+          },
+        },
+      },
```

Comment:
This is a counter because the value is cumulative

**See the panel with rate:**
![image](https://github.com/user-attachments/assets/564af95a-7fbb-4496-ac06-aa2b562e1fa1)
**and without:**
![image](https://github.com/user-attachments/assets/99139fe1-df7e-42db-87c8-e139830904dd)